administrator/language/en-GB/com_workflow.ini

```diff
@@ -35,7 +35,7 @@ COM_WORKFLOW_FORM_EDIT="Edit Workflow"
 COM_WORKFLOW_FORM_NEW="New Workflow"
```

Comment:
You should not fix "bugs" outside of your component scope. If you want to fix them, please go to the joomla-cms project and create an independent PR there.